pynsee/utils/_request_insee.py

```diff
@@ -44,10 +47,10 @@ def _request_insee(
     else:
         if pynsee_query_print == "True":
```

Comment:
we need to check what this ``pynsee_query_print`` is about: now that we're moving to logging, it's probably not needed anymore cc @hadrilec 

If we keep it, then switch between debug and warning depending on its value?